projects/demo/src/app/alert/angular/alert-angular-small.demo.html

```diff
@@ -13,12 +13,12 @@
     </header>
     <div class="content-container">
       <div class="content-area">
-        <clr-alert [clrAlertSizeSmall]="true">
+        <clr-alert [clrAlertSizeSmall]="true" [clrAlertLightweight]="true">
```

Comment:
```suggestion
        <clr-alert [clrAlertSizeSmall]="true" [clrAlertLightweight]="true" [clrAlertClosable]="false">
```

Reply:
I just removed my changes from the demo site, the demo app is not something we are supporting and it's basically not important to add something new there now.